README.md

```diff
@@ -61,6 +61,13 @@ Then you can run the deploy:
 $ ./deploy.sh
 ```
 
+Since this project already installs the `python-dotenv` package, you can do something like the following to run the deploy script with the correct environment variable values, if you want to avoid pasting credentials into your terminal and/or having them stored in your shell history:
+
+```shell
+# requires you to create a .env.qa file with the QA-specific env vars values needed by deploy.sh
+dotenv --file=.env.qa run ./deploy.sh
+```
```

Comment:
i thought this might be useful since i like to avoid pasting creds in my shell if possible, and since it might make it easier to keep credentials straight for stage vs QA.